CHANGELOG.md

```diff
@@ -8,12 +8,14 @@ CHANGELOG
 
 - Enable support for NICE DCV in GovCloud regions.
 - Enable support for AWS Batch scheduler in GovCloud regions.
+- Add ``-r/-region`` arg to ``pcluster configure``. If this arg is provided, configuration will skip region selection.
 
 **CHANGES**
 
 - Upgrade image used by CodeBuild environment when building container images for Batch clusters.
 - Enable queue resizing on update without requiring to stop the compute fleet. Stopping the compute fleet is only
   necessary when existing instances risk to be terminated.
+- Change the default of ``aws_region_name`` from us-east-1 to whatever region written in aws config file.
```

Comment:
The change should be something like:
- Remove default region us-east-1

Reply:
we could add also the "impact" from customer point of view, what does this change imply? what's needed to cope with this change?